proxy/src/project.rs

```diff
@@ -66,3 +71,13 @@ pub struct Stats {
     /// Amount of unique commiters on the default branch.
     pub contributors: u32,
 }
+
+/// TODO(xla): Add documentation.
+pub async fn get(paths: &librad::paths::Paths, id: &str) -> Result<Project, error::Error> {
+    let meta = coco::get_project_meta(paths, id)?;
+
+    Ok(Project {
+        id: librad::project::ProjectId::from_str(&id.to_string())?,
```

Comment:
you don't need to convert to string to then get a reference, you can use id directly: `librad::project::ProjectId::from_str(id)`

Reply:
Ha! Daily wtf. Thanks for the catch.